renderer/components/editor/options/left.js

```diff
@@ -56,7 +61,7 @@ class LeftOptions extends React.Component {
             width: 32px;
           }
 
-          input {
+          :global(input) {
```

Comment:
Are there no better ways to achieve this than using `:global()`?

Reply:
@sindresorhus, yeah it feels wrong to me, but I couldn't find a better way in the docs. I will dig in again to be sure there isn't a better way.

Reply:
@bringking Did you find anything better? Otherwise, I guess we can say it's good enough for now.

Reply:
@sindresorhus, I didn't dig in yet, sorry about that. I can carve some time for this tomorrow

Reply:
@sindresorhus I found the relevant docs for styled/jsx and got those globals cleaned up!

Reply:
Awesome!